packages/workflow/tsconfig.build.json

```diff
@@ -7,6 +7,6 @@
 		"noEmit": false,
 		"tsBuildInfoFile": "dist/build.tsbuildinfo"
 	},
-	"include": ["src/**/*.ts"],
+	"include": ["src/**/*.ts", "package.json"],
```

Comment:
why is this change needed?

Reply:
But CI is [flagging this](https://github.com/n8n-io/n8n/actions/runs/5143041094/jobs/9257556793#step:6:31) anyway so the fix must be elsewhere.